src/behavior-considered-undefined.md

```diff
@@ -51,7 +51,17 @@ code.
 * Invoking undefined behavior via compiler intrinsics.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
-* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI.
+* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI
+* Calling a foreign (e.g. C++) function that unwinds (`throw`s) via a function
+  declaration or pointer declared with a non-unwinding ABI such as `"C"`
```

Comment:
This is ambiguous: calling a function that *can* throw, or does the UB only occur if and when it *does* throw in a given execution?

It should be clarified to be the latter. Otherwise, a whole bunch of code that previously worked fine, with defined behaviour, would become UB by a strict reading. It is not obvious, because the foreign-language compiler might compile code that *can* throw differently.

Reply:
Yes, by "function that unwinds" I did mean "does" rather than "can"; but I think your phrasing of "causing an unwind to enter..." does seem clearer; thanks.